### DISCUSSION TITLE
Conquest

### DISCUSSION THREAD

**Discussion by Jigoku** [redeclipse/base#1213] [Maps and Content Showcase]
# Conquest

This map is themed around an ongoing war/battle on an earth-like planet in the Gliese system, features both a day/night variant. It's a symmetrically mirrored map, each with a team base.

The general length of the map is almost identical to Octavus. 
(ctf flag capture can be done in ~12 seconds, possibly less)

Map size: large (suitable for all game modes)

**Download**
~~r1037 - https://ufile.io/evmok9fj~~
~~r1131 - https://ufile.io/nv95uuh4~~
~~r1155 - https://ufile.io/etmsnkoy~~
~~r1186 - https://ufile.io/49qsdsiq~~
~~r1204 - https://ufile.io/ubaymmzq~~ 
**r1215 - https://ufile.io/7xob1eib <-- current version**
(~80mb due to additional textures)

Place the zip archive into your [user data directory](https://www.redeclipse.net/docs/FAQ#where-do-i-find-screenshots-logs-and-other-user-data),  no need to extract it. It should appear in the map selection menu.

### Notes (~~work in progress~~, possibly finished?)
 * requires latest master branch
 * ivy models are not vertex painted for wind
* there's some additional luckystrike/banner* models with the team logos / redeclipse logo, along with a modified mask (to make the colours less saturated)
* firefly model (night variant) is a reskin of the butterfly model, replacing that when `forcemapvariant 2` is set, it's very basic, but pulls off the effect as for how it's used with the glow channel in the mask.
 * performance seems almost similar to "rift" possibly a few fps less in areas, occlusion seems to be mostly ok until you're stood at the highest point of the map.
*  worth noting that the number of trees around the outer bounds of the map will impact fps a fair bit, if it's troublesome try setting `mapeffects 2` which removes shadows from the out of bounds trees and shadows from most mapmodels, or `mapeffects 1` which removes all of the out of bounds trees, as well as non-collidable mapmodels.

![2020-03-27-024420_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/77716161-661b3e00-6fd5-11ea-935c-f7d25860b8c9.png)

![2020-03-27-024431_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/77716167-69162e80-6fd5-11ea-97b4-aa4b1eeea290.png)

![2020-03-27-024439_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/77716173-6ca9b580-6fd5-11ea-95ee-4dcdab46409c.png)

![2020-03-27-024448_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/77716175-6f0c0f80-6fd5-11ea-9eb5-c7a687dc492d.png)

![2020-03-27-024526_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/77716182-759a8700-6fd5-11ea-9a3a-5e9f7ef39d74.png)

![2020-03-27-024534_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/77716190-7af7d180-6fd5-11ea-8ff0-e3e22e0e1ccb.png)

![2020-03-27-024624_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/77716192-7d5a2b80-6fd5-11ea-8fb7-441e727744bf.png)

![2020-03-27-024651_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/77716201-821edf80-6fd5-11ea-9c2f-7052b39ff5a4.png)

![2020-03-27-024659_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/77716203-83e8a300-6fd5-11ea-93e2-4b99a603d53b.png)

![2020-03-27-024513_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/77716206-8519d000-6fd5-11ea-94e5-2ba2fcaf9682.png)

**Comment by Jigoku** [comment]
- Updated download link, as for missing model changes

**Comment by dc-redeclipse** [comment]
Wow awsome! Echo looks like shit against that!

**Comment by Jigoku** [comment]
> Wow awsome! Echo looks like shit against that!

No it doesn't :laughing:

**Comment by Jigoku** [comment]
**Updated to r1131**

* texture fixes / vcolour changes
* weapon spawn adjustments
* lighting fixes
* more detail/randomisation to outer rock faces
* improve world/variant lighting
* vangle textures
* clip fixes
* optimise fxlevels for mapmodels

mapeffects 3 everything
mapeffects 2 removes most mapmodel shadows
mapeffects 1 removes non-collidable mapmodels

On my machine there's a 20fps increase each time when going from fxlevel 3 - > 2 - > 1
So it should be much better for lower spec systems now.

**Comment by Jigoku** [comment]
**Day variant**

![20200314002049](https://user-images.githubusercontent.com/1535179/76670946-59392c00-658b-11ea-8134-cafc8419f9ee.png)

**Night variant**

![20200314002044](https://user-images.githubusercontent.com/1535179/76670934-4cb4d380-658b-11ea-9735-63aa82631ee4.png)

**Comment by Jigoku** [comment]
**Updated to r1155**

* redo some waypoints
* many texture fixes
* change textures on some ceilings
* world lighting tweaks (blue tint)
* light entity tweaks
* move a few weapons about
* blendmap improvements
* add some more decals to rockfaces above team bases
* more mushrooms

![20200316021233](https://user-images.githubusercontent.com/1535179/76717565-cbe10d80-672b-11ea-939a-d7e64917047c.png)

![20200316021245](https://user-images.githubusercontent.com/1535179/76717569-d00d2b00-672b-11ea-9d55-7357c02c4a54.png)

**Comment by byronsharman** [comment]
Tells a strong story and looks beautiful!

**Comment by Jigoku** [comment]
**Updated to r1186**

* blendmap fixes
* lots of texture bugs/fixes
* tweak interior lighting to slightly more blue hue
* geometry fixes
* several decals repositioned / added
* improve pebbles blendmap on water boundary
* adjust weapon placement
* add parallaxed texture and particles to teleport area (instead of a door texture)
* added some volumetric lighting to the night variant

also

* lower mdlwind value for ivy models
  (not vertex painted)

**Comment by Jigoku** [comment]
**Update to r1204**

* misc texture fixes
* improve rails for the over head ships 
  * add sound entities
  * improve spawn placement when appearing from the horizon
* various light tweaks
* more blendmap adjustments

I consider this map finished for now, unless anyone can find any problems i've missed, or can suggest any small tweaks to improve it further.

Note; about the ivy models, i lowered the `mdlwind` so they have a smaller ripple effect, as vertex painting these became troublesome, and might not really be necessary?. There is a strange side-effect when scrolling/switching through weapons where the ivy meshes "shift" momentarily, not sure what can be done about that.

**Comment by Jigoku** [comment]
Also, may need some more play testing, as to check for affinity placements, and potentially whether weapons could be changed / moved for balance.

**Comment by Jigoku** [comment]
**Updated to r1211**
* texture fixes
* add env/skylump shader to window textures

**Comment by Jigoku** [comment]
**Updated to r1215**
* small texture fixes
* add a few more camera rails

**Comment by Jigoku** [comment]
[conquest-recolour.zip](https://github.com/redeclipse/world/files/5413316/conquest-recolour.zip)
Latest changes as for improving colour variation here, still need to fix the blendmap and potentially use some more flora models; I am also tempted to redo the outer trees, move them about a bit, and try to use less of them eventually.

Screens;

![2020-10-21-054220_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/96674528-ea158880-1360-11eb-934c-86dac3ed1606.png)
![2020-10-21-054337_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/96674515-e41fa780-1360-11eb-9b42-3e8f111c5d66.png)
![2020-10-21-054329_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/96674517-e550d480-1360-11eb-8de6-6bbb0f76b1a0.png)
![2020-10-21-054252_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/96674519-e6820180-1360-11eb-8f21-28597f204f1c.png)
![2020-10-21-054241_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/96674522-e7b32e80-1360-11eb-8381-f9d6451aaf90.png)
![2020-10-21-054232_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/96674525-e8e45b80-1360-11eb-9d48-8e3d8b159abe.png)
